PocketKit/Sources/Sync/PocketSaveService.swift

```diff
@@ -159,16 +159,20 @@ public class PocketSaveService: SaveService {
                 }
                 queue.addOperation(operation)
             } else {
-                let mutation = ReplaceSavedItemTagsMutation(input: [SavedItemTagsInput(savedItemId: remoteID, tags: names)])
+                let url = savedItem.item?.givenURL ?? savedItem.url
+                let mutation = SavedItemTagMutation(
+                    input: SavedItemTagInput(givenUrl: url, tagNames: names),
+                    timestamp: ISO8601DateFormatter.pocketGraphFormatter.string(from: .now)
```

Comment:
curious to hear about the context in behind adding this timestamp to `tag` and why it's needed as an input

Reply:
@cyndichin is it fair to say "because the backend says so"? 😅